fenix/app/src/release/res/xml/shortcuts.xml

```diff
@@ -3,6 +3,17 @@
    - License, v. 2.0. If a copy of the MPL was not distributed with this
    - file, You can obtain one at http://mozilla.org/MPL/2.0/. -->
 <shortcuts xmlns:android="http://schemas.android.com/apk/res/android">
+    <shortcut
+        android:shortcutId="password_manager"
+        android:enabled="true"
+        android:icon="@drawable/ic_static_shortcut_password_manager"
```

Comment:
Same issue here. This should be `android:icon="@drawable/ic_static_password_shortcut"`